pkg/upgrade/control_plane.go

```diff
@@ -616,6 +616,18 @@ func (u *ControlPlaneUpgrader) updateMachines(machines []*clusterv1.Machine) err
 
 		// TODO skip if the bootstrap ref is not a KubeadmConfig
 
+		log.Info("Checking etcd health")
+		err := wait.PollImmediate(10*time.Second, 5*time.Minute, func() (bool, error) {
+			if err := u.etcdClusterHealthCheck(15 * time.Second); err != nil {
```

Comment:
In `etcdClusterHealthCheck`, could we use the `--cluster` flag instead of `--endpoints` now, since we are no longer supporting v1.13?

Otherwise, lgtm

Reply:
I think I'd feel more comfortable leaving the `--endpoints` bit for now, since we know it works.

Reply:
Sounds good, should I create an issue to track following up or are we okay with leaving it as is, since v1alpha3 should obviate the need for this tool?

Reply:
I'm ok leaving as-is w/v1a3 looming